core/control-panel/impl/src/core/config.rs

```diff
@@ -37,7 +37,7 @@ impl Default for CanisterConfig {
             station_wasm_module: vec![],
             station_wasm_module_extra_chunks: None,
             last_upgrade_timestamp: time(),
-            version: None,
+            version: Some(SYSTEM_VERSION.to_string()),
```

Comment:
was this a bug that this was missing?

Reply:
I believe this is a bug that doesn't matter in production as we don't deploy new control panels from scratch.

Reply:
also in prod we always use the `CanisterConfig::new(args)` instead of `CanisterConfig::default()`, and new does include the version.